jwst/photom/photom.py

```diff
@@ -328,11 +324,9 @@ def calc_nirspec(self, ftab, area_fname):
                 # on its wavelength
                 sens2d = np.interp(wave2d, waves, relresps)
                 sens2d *= tabdata['photmj']  # include the initial scalar conversion factor -> MJ
-                # This line used to be applied to all IFU data, but I think this was an error -
-                # masked by the fact that the initial pixel area maps for IFU data were
-                # arrays of 1-values (in arcsec**2). If srctype==POINT, do not apply.
-                if self.source_type.upper() != 'POINT':
-                    sens2d /= area2d * A2_TO_SR  # divide by pixel area * A2_TO_SR -> MJ/sr
+                # Apply to all data (both point source and extended)
```

Comment:
Instead of just saying "Apply", which leaves open the question of "apply what and why?" make this comment more explicit, e.g. "Convert all data (both point source and extended) to surface brightness"